### PR TITLE
Registration form translations

### PR DESCRIPTION
**Pull request by aryansri-19**
- Some sentences are left like the error messages which are custom.

### CODE REVIEW

File: src/pages/Registration/MemberForm.jsx [hunk 1]
```diff
@@ -90,11 +90,11 @@ const MemberForm = ({ control, index }) => {
...
             <Select {...genderField} required labelId="gender" label="Gender">
-              <MenuItem value="Male">Male</MenuItem>
-              <MenuItem value="Female">Female</MenuItem>
-              <MenuItem value="Other">Others</MenuItem>
+              <MenuItem value="Male">{t("male")}</MenuItem>
+              <MenuItem value="Female">{t("female")}</MenuItem>
+              <MenuItem value="Other">{t("others")}</MenuItem>
             </Select>
           </FormControl>
           <p error={genderFieldState.invalid} style={{ color: "red" }}>
```
Comment (anvbey) [inline src/pages/Registration/MemberForm.jsx:100]: Fix the custom error messages also

File: src/pages/Registration/GroupLeaderForm.jsx [hunk 2]
```diff
@@ -256,17 +257,17 @@ const GroupForm = () => {
...
               >
-                <MenuItem value="Male">Male</MenuItem>
-                <MenuItem value="Female">Female</MenuItem>
-                <MenuItem value="Other">Others</MenuItem>
+                <MenuItem value="Male">{t("male")}</MenuItem>
+                <MenuItem value="Female">{t("female")}</MenuItem>
+                <MenuItem value="Other">{t("others")}</MenuItem>
               </Select>
             </FormControl>
             <p error={!!errors.gender} style={{ color: "red" }}>
```
Comment (anvbey) [inline src/pages/Registration/GroupLeaderForm.jsx:273]: Fix needed here

File: src/pages/Registration/MemberForm.jsx (hunk 1 above)
Comment (aryansri-19) [reply src/pages/Registration/MemberForm.jsx:100]: These errors are of react-hook-form. Cannot translate them

Can only translate the ones in zod validation

Comment (anvbey) [reply src/pages/Registration/MemberForm.jsx:100]: What kind of errors does this throw mainly?

Comment (aryansri-19) [reply src/pages/Registration/MemberForm.jsx:100]: Caused by incorrect filling of form according to rules of hook-form. Like NaN value, empty value or can be something else

Comment (anvbey) [reply src/pages/Registration/MemberForm.jsx:100]: Got it so these are the errors which one must understand 
If the errors can't be shown in Hindi then there's no point of translation
See if something can be done about it

**Pull request merged**